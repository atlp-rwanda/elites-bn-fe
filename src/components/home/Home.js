<<<<<<< HEAD
import React from 'react';

const Home = () => (
  <div className="App">
    <header className="App-header">
      <p>Welcome to Barefoot</p>
    </header>
  </div>
);
export default Home;
=======
import React, { useEffect } from 'react';
import { makeStyles } from '@mui/styles';
import Typography from '@mui/material/Typography';
import Container from '@mui/material/Container';
import { easings } from 'react-animation';
import 'animate.css';
import NavBar from './WelcomeNavBar';
import background from '../../Assets/Images/background.jpg';
import Bottom from './BottomNavBar.js';

const useStyles = makeStyles({
  home: {
    backgroundImage:`url(${background})`,
    height: '100vh',
    backgroundSize: '100% 100%',
    backgroundRepeat: 'no-repeat',
  },

  centerDiv: {
    textAlign: 'center',
    margin: 'auto',
    padding: '20vh',
  },

  animate: {
    animation: `bounce-in-up  ${easings.easeOutCubic} 1000ms forwards`,
  },
});


const HomeComponent = () => {
  useEffect(() => {
    document.body.style.overflow = 'hidden';
  }, []);
  const classes = useStyles();
  return (
    <div className={classes.home}>
        <NavBar/>
      <Container className={classes.centerDiv}>
        <Typography 
        variant="h2" 
        gutterBottom
        className="animate__animated animate__bounceInRight"
        style={{ color:'white'}}
        >
          Welcome to Barefoot Nomad
        </Typography>

        <Typography
          variant="h3"
          gutterBottom
          className="animate__animated animate__bounceInLeft"
          style={{ color:'#D5ECD1'}}
        >
          Travel without Boundaries
        </Typography>
        <hr className="animate__animated animate__bounceInRight" style={{color:'black'}}/>
      </Container>
      <Bottom />
    </div>
  );
};

export default HomeComponent;
>>>>>>>  This is a combination of 10 commits.
