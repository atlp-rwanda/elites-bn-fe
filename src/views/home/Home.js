import React from 'react';
import Typography from '@mui/material/Typography';
import Container from '@mui/material/Container';
import { useTranslation} from 'react-i18next';
import 'animate.css';
import NavBar from './WelcomeNavBar';
import Bottom from './BottomNavBar.js';
import useStyles from './homeStyle';

const Home = () => {
  const { t } = useTranslation();
  const classes = useStyles();
  return (
    <div className={classes.home}>
      <NavBar />
      <Container className={classes.centerDiv}>
        <Typography
          variant="h2"
          gutterBottom
          className="animate__animated animate__bounceInRight"
          style={{ color: 'white' }}
        >
          {t('Welcome to Barefoot Nomad')}
        </Typography>

        <Typography
          variant="h3"
          gutterBottom
          className="animate__animated animate__bounceInLeft"
          style={{ color: '#D5ECD1' }}
        >
          {t('Travel without Boundaries')}
        </Typography>
        <hr
          className="animate__animated animate__bounceInRight"
          style={{ color: 'black' }}
        />
      </Container>
      <Bottom />
    </div>
  );
};
export default Home;
