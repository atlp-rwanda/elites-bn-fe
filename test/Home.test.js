import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Home from '../src/components/home/Home';

describe('Snapshot Test', () => {
  test('Snapshot for Home Component', () => {
    const renderedComponent = renderer.create(<Home />).toJSON();
    expect(renderedComponent).toMatchSnapshot();
  });
});
