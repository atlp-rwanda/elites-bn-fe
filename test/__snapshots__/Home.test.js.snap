// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test Snapshot for Home Component 1`] = `

    <h2>
      Welcome to Barefoot
    </h2>
 
`;
